src/main/java/ProjectLibraryPartTwo/DAO/SQLLibraryDAO.java

```diff
@@ -0,0 +1,199 @@
+package ProjectLibraryPartTwo.DAO;
+
+import ProjectLibraryPartTwo.Connection.ConnectorBD;
+import ProjectLibraryPartTwo.Entity.Author;
+import ProjectLibraryPartTwo.Entity.Book;
+
+import java.sql.*;
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+import java.util.concurrent.TimeUnit;
+
+public class SQLLibraryDAO implements LibraryDAO {
```

Comment:
we write package name in lower case. There are better names LibraryDao and LibraryDaoImpl. But also better create these methods in BookDao, because we work wih book entity

Reply:
fixed